### ISSUE TITLE
View Presenters

### ISSUE THREAD

**Issue by keeross** [adonisjs/docs#258]
Should we add some docs on on view presenters? It is a pretty useful thing that is not well documented at the moment.

**Comment by thetutlage** [comment]
They are covered in the edge docs http://edge.adonisjs.com/docs/presenters

**Comment by keeross** [comment]
Yes, but there is no default view presenter example, you have to watch full video in order to get it.
I think it has to be in Adonis docs, as soon as it is more relevant to it, at the same time there should be `adonis make:presenter command` so people can generate them properly and know where to place them.
I can create a PR into Adonis or Edge with command and docs.
